snoop/data/management/commands/filestats.py

```diff
@@ -0,0 +1,128 @@
+import sys
+from django.core.management.base import BaseCommand
+from ...logs import logging_for_management_command
+from ... import models
+from ... import collections
+from django.db.models import Count
+from django.db.models import Sum
+import re
+import mimetypes
+
+SUPPORTED_FILETYPES = {
```

Comment:
Actually, I think you can re-create this whole list from the code base itself. Just import all of the respective modules and concatenate the lists inside. For places where the content type isn't extracted as a global constant string or list, then refactor it and add a reference to it in this file.

This will let us maintain this command in the long term without actually visiting it.